go-controller/pkg/node/ovn_test.go

```diff
@@ -53,11 +49,11 @@ func (o *FakeOVNNode) start(ctx *cli.Context, objects ...runtime.Object) {
 	}
 	_, err := config.InitConfig(ctx, o.fakeExec, nil)
 	Expect(err).NotTo(HaveOccurred())
-
-	o.fakeCRDClient = apiextensionsfake.NewSimpleClientset()
-	o.fakeEgressClient = egressfirewallfake.NewSimpleClientset()
-	o.fakeClient = fake.NewSimpleClientset(v1Objects...)
-	o.fakeEgressIPClient = egressipfake.NewSimpleClientset(egressIPObjects...)
+	o.fakeClient = &util.OVNClientset{
+		KubeClient:          fake.NewSimpleClientset(v1Objects...),
+		EgressIPClient:      egressipfake.NewSimpleClientset(egressIPObjects...),
+		APIExtensionsClient: apiextensionsfake.NewSimpleClientset(),
+	}
```

Comment:
you dropped the egressfirewall client?

Reply:
On purpose. Egress firewall does not perform any operation on the node side, so no need to instantiate its client set.